BookingsApi/BookingsApi.DAL/Commands/AddJusticeUserCommand.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Exceptions;
+using BookingsApi.Domain;
+using BookingsApi.Domain.Enumerations;
+using BookingsApi.Domain.RefData;
+
+namespace BookingsApi.DAL.Commands
+{
+    public class AddJusticeUserCommand : ICommand
+    {
+        public string FirstName { get; }
+        public string Lastname { get; }
+        public string Username { get; }
+        public string ContactEmail { get; }
+        public string Telephone { get; set; }
+        public string CreatedBy { get; }
+        public int RoleId { get; }
+
+        public AddJusticeUserCommand(string firstName, string lastname, string username, string contactEmail,
+            string createdBy, int roleId)
+        {
+            FirstName = firstName;
+            Lastname = lastname;
+            Username = username;
+            ContactEmail = contactEmail;
+            CreatedBy = createdBy;
+            RoleId = roleId;
+        }
+    }
+
+    public class AddJusticeUserCommandHandler : ICommandHandler<AddJusticeUserCommand>
+    {
+        private readonly BookingsDbContext _context;
+
+        public AddJusticeUserCommandHandler(BookingsDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task Handle(AddJusticeUserCommand command)
+        {
+            var roleName = command.RoleId == (int)UserRoleId.VhTeamLead ? "Video hearings team lead" : "Video hearings officer";
```

Comment:
Assume this is for the label we display in the u.i - might want to double check with Lorenzo on the naming. If you have already, no worries you can resolve this comment. But I think he said before, they want to be called CSO now rather than a VHO

Reply:
is that perhaps the existing role name in the database, so that it can do the lookup?

Reply:
is that perhaps the existing role name in the database, required so that it can find the matching user role object? If so, we would probably need to leave the role name as is

Reply:
^ you're right. Ignore me